openassessment/xblock/xml.py

```diff
@@ -760,7 +761,7 @@ def serialize_examples_to_xml_str(assessment):
         A unicode string of the XML serialized examples.
 
     """
-    examples = assessment.get('examples', [])
+    examples = update_assessments_format([assessment])[0].get('examples', [])
```

Comment:
@symbolist i can't understand why we are choosing 0th element. can you please explain?